app/models.py

```diff
@@ -2142,3 +2147,65 @@ def serialize(self):
             "created_at": created_at_in_bst.strftime("%Y-%m-%d %H:%M:%S"),
         }
         return contact_list
+
+
+class BroadcastStatusType(db.Model):
+    __tablename__ = 'broadcast_status_type'
+    DRAFT = 'draft'
+    PENDING_APPROVAL = 'pending-approval'
+    REJECTED = 'rejected'
+    BROADCASTING = 'broadcasting'
+    COMPLETED = 'completed'
+    CANCELLED = 'cancelled'
+    TECHNICAL_FAILURE = 'technical-failure'
+
+    STATUSES = [DRAFT, PENDING_APPROVAL, REJECTED, BROADCASTING, COMPLETED, CANCELLED, TECHNICAL_FAILURE]
+
+    name = db.Column(db.String, primary_key=True)
+
+
+class BroadcastMessage(db.Model):
+    __tablename__ = 'broadcast_message'
+    __table_args__ = (
+        db.ForeignKeyConstraint(
+            ['template_id', 'template_version'],
+            ['templates_history.id', 'templates_history.version'],
+        ),
+        {}
+    )
+
+    id = db.Column(UUID(as_uuid=True), primary_key=True)
+
+    service_id = db.Column(UUID(as_uuid=True), db.ForeignKey('services.id'))
+    service = db.relationship('Service', backref='broadcast_messages')
+
+    template_id = db.Column(UUID(as_uuid=True), nullable=False)
+    template_version = db.Column(db.Integer, nullable=False)
+    template = db.relationship('TemplateHistory', backref='broadcast_messages')
+
+    _personalisation = db.Column(db.String, nullable=True)
```

Comment:
Will _personalisation only be used for place holders in the template?

Reply:
Yup, that's my intention. There _may_ be some mandatory place holders (eg, for things like title), but we'll cross that bridge when we understand more about what the networks require

Reply:
I hope we don't need mandatory place holders